app/models/event.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class Event < ApplicationRecord
+  belongs_to :category
+  belongs_to :user
+  validates :name, presence: true
+  validates :name, length: { minimum: 3 }
+  validates :scheduled_for, presence: true
+  validate :scheduled_for_after_now?
```

Comment:
Please split code into logical parts using new lines. F.e. associations, default validations and custom validations should be split by new lines.
```suggestion
  belongs_to :category
  belongs_to :user
  
  validates :name, presence: true
  validates :name, length: { minimum: 3 }
  validates :scheduled_for, presence: true
  
  validate :scheduled_for_after_now?
```